README.md

```diff
@@ -1 +1,5 @@
 # canvasCourseDeleteUtility
+
```

Comment:
This is not correctly formatted for MarkDown.

This says "Build instructions" but they are both build and run instructions.

Why not have a build.sh?  That is easier to use on the build server.

Tool configuration should be documented.